client/rpc/src/state/mod.rs

```diff
@@ -266,6 +271,10 @@ impl<Block, Client> StateApi<Block::Hash> for State<Block, Client>
 		key_prefix: StorageKey,
 		block: Option<Block::Hash>,
 	) -> FutureResult<Vec<StorageKey>> {
+		if let Err(e) = self.deny_unsafe.check_if_safe() {
```

Comment:
These ones (the whole `state` module) are currently used on our public RPC node, and I think they should not be marked. While they can generate high load on the machine, I think we should keep the flag to thing that are more insecure than simply DDoS vector (for instance we did left out `author_submitExtrinsic` (Which is correct)).

Reply:
Makes sense, unmarked the entire module.